modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -0,0 +1,842 @@
+====
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `remote-with-driver-settings` FOR DATABASE `northwind-graph-2020`
+AT "neo4j+s://location:7687"
+USER alice
+PASSWORD 'example_secret'
+DRIVER {
+  connection_timeout: duration({minutes: 1}),
+  connection_pool_max_size: 10
+}
+----
+
+When a database alias pointing to a remote database has been created, its details can be shown with the `SHOW ALIASES FOR DATABASE` command.
+
+.Query
+[source, cypher]
+----
+SHOW ALIAS `remote-with-driver-settings` FOR DATABASE YIELD *
+----
+
+.Result
+[role="queryresult"]
+----
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| name                          | composite | database               | location | url                       | user    | driver                                                   | properties |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "remote-with-driver-settings" | NULL      | "northwind-graph-2020" | "remote" | "neo4j+s://location:7687" | "alice" | {connection_pool_max_size: 10, connection_timeout: PT1M} | {}         |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+
+==== Set properties for remote database aliases
+
+Just as the local database aliases, the remote database aliases can be given properties.
+These properties can then be used in queries with the xref::database-administration/composite-databases/querying-composite-databases.adoc#composite-databases-queries-built-in-functions[`graph.propertiesByName()` function].
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `remote-northwind-2021` FOR DATABASE `northwind-graph-2021` AT 'neo4j+s://location:7687'
+USER alice PASSWORD 'password'
+PROPERTIES { newestNorthwind: false, index: 6 }
+----
+
+The properties are then shown in the `SHOW ALIASES FOR DATABASE YIELD ...` command.
+
+.Query
+[source, cypher]
+----
+SHOW ALIAS `remote-northwind-2021` FOR DATABASE YIELD name, properties
+----
+
+.Result
+[role="queryresult"]
+----
++--------------------------------------------------------------+
+| name                    | properties                         |
++--------------------------------------------------------------+
+| "remote-northwind-2021" | {index: 6, newestnorthwind: FALSE} |
++--------------------------------------------------------------+
+----
+
+[NOTE]
+====
+For more information on  local and remote database aliases in a composite database, see xref::database-administration/composite-databases/manage-composite-databases.adoc#create-composite-database-alias[Create database aliases in composite databases].
+====
```

Comment:
this doesn't feel like it belongs here?

Reply:
Agreed - I don't think the note is necessary at all anymore with the new setup (it is already clear enough as it has its own subchapter)